addons/account/views/ir_module_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="account_view_module_filter" model="ir.ui.view">
```

Comment:
Regarding the guidelines, it should be `view_module_filter_inherit_account` and `name` should be `ir.module.module.list.inherit.account`

Reply:
Thanks, I'll make sure to go re-read the guidelines for these points!